test1-prod.js

```diff
@@ -7,4 +7,6 @@ if(fooProd  === "bar") {
   console.log( 'sha from event' )
   
   console.log( 'sha checking event name for real with logs' )
+  
+  console.log( 'this PR will trigger analyis for the PR' )
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/troubleshoot-codacy/eslint-test-prod/pullRequest?prid=7465056)
```suggestion
  console.log( "this PR will trigger analyis for the PR" )
```